server/src/main/java/org/candlepin/util/Util.java

```diff
@@ -107,19 +107,19 @@ public static String generateDbUUID() {
     }
 
     public static <E> List<E> newList() {
-        return new ArrayList<E>();
+        return new ArrayList<>();
```

Comment:
This actually works...??

Reply:
I'm assuming that the compiler can infer the type from the method signature?  But why do we even have that method.  Seems like a waste of time and it's only referenced from unit test classes.

Reply:
Definitely -- especially the list-based ones where Collections and Arrays natively provide better functionality.